tests/embedders/geovex/test_embedder.py

```diff
@@ -0,0 +1,83 @@
+"""Hex2VecEmbedder tests."""
+from contextlib import nullcontext as does_not_raise
+from pathlib import Path
+from typing import Any, Dict, List, Union
+
+import geopandas as gpd
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+from pytorch_lightning import seed_everything
+
+from srai.embedders.geovex.embedder import GeoVexEmbedder
+from srai.exceptions import ModelNotFitException
+from srai.loaders.osm_loaders.filters import HEX2VEC_FILTER
+from srai.neighbourhoods import H3Neighbourhood
+from tests.embedders.geovex.constants import EMBEDDING_SIZE, PREDEFINED_TEST_CASES, TRAINER_KWARGS
+
+
+@pytest.mark.parametrize(  # type: ignore
+    "target_features,expectation",
+    [
+        (None, does_not_raise()),
+        (["building"], pytest.raises(ValueError)),
+    ],
+)
+def test_checking_encoder_sizes(target_features: Union[str, List[str]], expectation: Any) -> None:
+    """Test that incorrect encoder sizes raise ValueError."""
+    target_tags: Dict[str, List[str]] = (
+        target_features if target_features else HEX2VEC_FILTER  # type: ignore
+    )
+    target_features = [f"{t}_{st}" for t in target_tags for st in HEX2VEC_FILTER[t]]  # type: ignore
+
+    neighbourhood = H3Neighbourhood()
+    with expectation:
+        GeoVexEmbedder(
+            target_features=target_features,
+            neighbourhood=neighbourhood,
+        )
+
+
+def test_embedder_not_fitted() -> None:
+    """Test that Hex2VecEmbedder raises ModelNotFitException if not fitted."""
+    embedder = GeoVexEmbedder(
+        [f"{t}_{st}" for t in HEX2VEC_FILTER for st in HEX2VEC_FILTER[t]],  # type: ignore
+        H3Neighbourhood(),
+    )
+    with pytest.raises(ModelNotFitException):
+        embedder.transform(gpd.GeoDataFrame(geometry=[]), gpd.GeoDataFrame(), gpd.GeoDataFrame())
+
+
+def test_embedder() -> None:
```

Comment:
Missing the files for this. The pre-commit hook `check-added-large-files` prevented adding them. Can't get the `*_features.parquet` small enough with the >= 256 features requirement.

Reply:
Do you know how big was this file? We have increased it to 1mb from the default 500kb for another use case.

Reply:
The features file is 22MB...

I will make the conv in-channels a parameter (which it should be anyways), then fitting inside the 1MB limit will be do-able

Reply:
[29bd6cf](https://github.com/kraina-ai/srai/pull/293/commits/29bd6cff08cade7b1e08c33d0bd7064cac778370) fits into the 1MB requirement